### ISSUE TITLE
Linux make error

### ISSUE THREAD

**Issue by sudhevan** [zrax/pycdc#229]
Hi,
generated make file with 
`cmake -S <input_dir> -B <output_dir>`

trying to build on Linux
same error with python3.9&3.10, ami missing something?

```
make
[  2%] Generating bytes/python_10.cpp, bytes/python_11.cpp, bytes/python_13.cpp, bytes/python_14.cpp, bytes/python_15.cpp, bytes/python_16.cpp, bytes/python_20.cpp, bytes/python_21.cpp, bytes/python_22.cpp, bytes/python_23.cpp, bytes/python_24.cpp, bytes/python_25.cpp, bytes/python_26.cpp, bytes/python_27.cpp, bytes/python_30.cpp, bytes/python_31.cpp, bytes/python_32.cpp, bytes/python_33.cpp, bytes/python_34.cpp, bytes/python_35.cpp, bytes/python_36.cpp, bytes/python_37.cpp, bytes/python_38.cpp, bytes/python_39.cpp, bytes/python_310.cpp
[  4%] Building CXX object CMakeFiles/pycxx.dir/bytecode.cpp.o
In file included from pycdc-master/pyc_numeric.h:4:0,
                 from pycdc-master/bytecode.cpp:1:
pycdc-master/pyc_object.h: In constructor ‘PycObject::PycObject(int)’:
pycdc-master/pyc_object.h:132:40: error: declaration of ‘type’ shadows a member of 'this' [-Werror=shadow]
     PycObject(int type = TYPE_UNKNOWN) : m_refs(0), m_type(type) { }
                                        ^
In file included from pycdc-master/bytecode.cpp:1:0:
pycdc-master/pyc_numeric.h: In constructor ‘PycInt::PycInt(int, int)’:
pycdc-master/pyc_numeric.h:12:9: error: declaration of ‘type’ shadows a member of 'this' [-Werror=shadow]
         : PycObject(type), m_value(value) { }
         ^
pycdc-master/pyc_numeric.h:12:9: error: declaration of ‘value’ shadows a member of 'this' [-Werror=shadow]
pycdc-master/pyc_numeric.h: In constructor ‘PycLong::PycLong(int)’:
pycdc-master/pyc_numeric.h:31:9: error: declaration of ‘type’ shadows a member of 'this' [-Werror=shadow]
         : PycObject(type), m_size(0) { }

```

**Comment by zrax** [comment]
I can't reproduce this with GCC 11 or Clang 13...  What compiler are you using?  Do you have any other local changes?

**Comment by sudhevan** [comment]
My bas, was using gcc 4.8.5
after updating to gcc7, build worked fine. but pycdc seems to fail on conversion

getting this warning after printing ~50 lines of code

**# WARNING: Decompyle incomplete**

**Comment by zrax** [comment]
Fixed with #469
